docs/_headers/license-prepend-missing.sh

```diff
@@ -2,7 +2,7 @@
 #
 # Copyright (c) 2017, UT-BATTELLE, LLC
 # All rights reserved.
-# 
+#
```

Comment:
@kevans32, sorry about all these things.  My editor automatically deletes trailing white space, which is nice sometimes but not in this case. I can revert these if they are annoying to you.